src/Bridge/Doctrine/Orm/Filter/OrderFilter.php

```diff
@@ -143,6 +143,11 @@ protected function filterProperty(string $property, $direction, QueryBuilder $qu
      */
     protected function extractProperties(Request $request/*, string $resourceClass*/): array
     {
-        return $request->query->get($this->orderParameterName, []);
+        $properties = $request->query->get($this->orderParameterName, []);
+        if (!is_array($properties)) {
+            return [];
```

Comment:
If the parameter is not an array (ie only ?order=foobar), PHP throws a type error because the returned value is a string, not an array